p2p/src/network_ext.rs

```diff
@@ -0,0 +1,89 @@
+//! SPDX-License-Identifier: CC0-1.0
+//!
+//! This module implements an extension trait for [`Network`]
+//! with getter methods for the default P2P port and default
+//! network [`Magic`] bytes.
```

Comment:
If we make the module private it doesn't matter too much but perhaps this is cleaner (at least we should not have the SPDX in the rustdoc)

```rust
// SPDX-License-Identifier: CC0-1.0

//! This module implements an extension trait for [`Network`] with getter methods for the default
//! P2P port and default network [`Magic`] bytes.
//!
//! [`Network`]: bitcoin::Network
//! [`Magic`]: crate::Magic
```